controllers/users.js

```diff
@@ -70,6 +70,31 @@ const getUsers = async (req, res) => {
     const query = req.query?.query ?? "";
     const qualifiers = getQualifiers(query);
 
+    // getting user details by id if present.
+    if (req.query.id) {
+      const id = req.query.id;
+      let result;
+      try {
+        result = await userQuery.fetchUser({ userId: id });
+      } catch (error) {
+        logger.error(`Error while fetching user: ${error}`);
+        return res.boom.serverUnavailable(SOMETHING_WENT_WRONG);
+      }
+
+      if (!result.userExists) {
+        return res.boom.notFound("User doesn't exist");
+      }
+
+      const user = { ...result.user };
+      delete user.phone;
+      delete user.email;
```

Comment:
I think we should remove token and chain code also
REF https://github.com/Real-Dev-Squad/website-backend/blob/a4952edc866ccbb6d9a8645481e9304e67adc4ae/models/users.js#L167-L174

Reply:
okay

Reply:
> I think we should remove token and chain code also
> REF

You asked so, I removed them